src/Components/AddItem.jsx

```diff
@@ -1,5 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import firebase from 'firebase';
+import { db } from '../lib/firebase';
 
 export default function AddItem() {
-  return <div>Hi from Add an Item View!</div>;
+  const [token, setToken] = useState('test-token');
+  const { register, handleSubmit, watch, errors } = useForm();
+  const onSubmit = (data) => {
+    const currentList = db.collection('lists').doc(token);
+    currentList
+      .update({
+        items: firebase.firestore.FieldValue.arrayUnion({
+          name: data.itemName,
+          frequency: parseInt(data.itemFrequency),
+          lastPurchased: null,
+        }),
+      })
+      .then(() => alert('Item has been submitted!'));
+  };
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <br />
```

Comment:
Also not a blocker, but are these `<br>` tags temporary styling for this issue?

Reply:
Yes, the breaks were used for temporary styling so that the app looked a little better :)